messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/HTTPAcceptorAdd.java

```diff
@@ -55,14 +55,14 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
     void launchServices(OperationContext context, String activeMQServerName, String acceptorName, ModelNode model) throws OperationFailedException {
         String httpConnectorName = HTTPAcceptorDefinition.HTTP_LISTENER.resolveModelAttribute(context, model).asString();
 
-        HTTPUpgradeService.installService(context.getServiceTarget(),
+        HTTPUpgradeService.installService(context,
                 activeMQServerName,
                 acceptorName,
                 httpConnectorName);
 
         boolean upgradeLegacy = HTTPAcceptorDefinition.UPGRADE_LEGACY.resolveModelAttribute(context, model).asBoolean();
         if (upgradeLegacy) {
-            HTTPUpgradeService.LegacyHttpUpgradeService.installService(context.getServiceTarget(),
+            HTTPUpgradeService.LegacyHttpUpgradeService.installService(context,
```

Comment:
`context.getServiceTarget()` should be replaced by `context.getCapabilityServiceTarget()`